src/MainWindow.vala

```diff
@@ -1229,8 +1228,28 @@ namespace Terminal {
                 }
                 var text = intext.strip ();
 
+                /* warn on pasting text with `sudo` */
                 if ((text.index_of ("sudo") > -1) && (text.index_of ("\n") != 0)) {
-                    var dialog = new UnsafePasteDialog (this, text);
+                    var dialog = new UnsafePasteDialog (
+                        this,
+                        _("This command is asking for Administrative access to your computer"),
```

Comment:
Strictly speaking, the word "sudo" might be inside quotes in the pasted text (which may not be a command). Maybe rethink this text?

Reply:
This copy is exactly as-is from master, but I can rephrase to `may` to match the multi-line paste text.